src/boot/axios.ts

```diff
@@ -19,17 +20,34 @@ const isUnprotected = (url: string): boolean => {
 
 fusionAuthApi.interceptors.request.use(
   (config: InternalAxiosRequestConfig) => {
+    Loading.show();
     if (!isUnprotected(config.url || '')) {
       const token = sessionStorage.getItem(AUTH_TOKEN_NAMES.ACCESS_TOKEN);
       if (token) {
         config.headers.Authorization = `Bearer ${token}`;
       }
-    } else {
-      console.warn('No token found for protected endpoint.');
     }
     return config;
   },
   error => Promise.reject(error)
 );
 
+// Add a response interceptor
+fusionAuthApi.interceptors.response.use(
+  response => {
+    Loading.hide();
+    return response;
+  },
+  error => {
+    Loading.hide();
+    if (error.config && error.config.method === 'post') {
+      Notify.create({
+        message: error.response?.data.error,
```

Comment:
Awesome implementation, love that you took the initiative to work on error messages right away. As spoken today, let's check here if `error.response?.data.error` is an empty string, if so, display a standard HTTP Status Text. 

I suggest using [this documentation](https://developer.mozilla.org/en-US/docs/Web/API/Response/statusText).

As a bonus, you could prefix the error message with the `HTTPSTATUS` of the response. This way, the "message" attribute of this component would show **both the status of the response, and the message associated to it.** 

E.g:
- "404: Unauthorized"
- (or a very ugly one) "400: applicationId: Invalid JSON in the request body. The property was [applicationId]. The error was [Possible conversion error]. The detailed exception was [Cannot deserialize value of type `java.util.UUID` from String "c412a5bf-524-46e9-85a6-08d1f1777295": UUID has to be represented by standard 36-char representation at [Source: (io.fusionauth.http.io.ReaderBlockingByteBufferInputStream); line: 1, column: 46] (through reference chain: io.fusionauth.domain.api.LoginRequest["applicationId"])]."